gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.14-bin.zip
```

Use use latest gradle by updating the plugin:
```kts
gmazzo-test-aggregation = { id = "io.github.gmazzo.test.aggregation.results", version = "2.4.4" }
```